? bk-deps
? mypatch.patch
? version-script
? wxGenericDirCtrl_031227.patch
? wxgtk-2.5-config
? contrib/Makefile
? contrib/samples/Makefile
? contrib/samples/animate/Makefile
? contrib/samples/applet/Makefile
? contrib/samples/deprecated/Makefile
? contrib/samples/deprecated/proplist/Makefile
? contrib/samples/deprecated/resource/Makefile
? contrib/samples/deprecated/treelay/Makefile
? contrib/samples/fl/Makefile
? contrib/samples/gizmos/Makefile
? contrib/samples/gizmos/dynsash/Makefile
? contrib/samples/gizmos/dynsash_switch/Makefile
? contrib/samples/gizmos/editlbox/Makefile
? contrib/samples/gizmos/led/Makefile
? contrib/samples/gizmos/multicell/Makefile
? contrib/samples/gizmos/splittree/Makefile
? contrib/samples/mmedia/Makefile
? contrib/samples/ogl/Makefile
? contrib/samples/ogl/ogledit/Makefile
? contrib/samples/ogl/studio/Makefile
? contrib/samples/plot/Makefile
? contrib/samples/stc/Makefile
? contrib/samples/svg/Makefile
? contrib/samples/xrc/Makefile
? contrib/src/Makefile
? contrib/src/animate/Makefile
? contrib/src/applet/Makefile
? contrib/src/deprecated/Makefile
? contrib/src/fl/Makefile
? contrib/src/gizmos/Makefile
? contrib/src/mmedia/Makefile
? contrib/src/net/Makefile
? contrib/src/ogl/Makefile
? contrib/src/plot/Makefile
? contrib/src/stc/Makefile
? contrib/src/svg/Makefile
? contrib/src/xrc/Makefile
? contrib/utils/Makefile
? contrib/utils/convertrc/Makefile
? contrib/utils/wxrc/Makefile
? contrib/utils/wxrcedit/Makefile
? demos/Makefile
? demos/bombs/Makefile
? demos/dbbrowse/Makefile
? demos/forty/Makefile
? demos/fractal/Makefile
? demos/life/Makefile
? demos/poem/Makefile
? lib/wx
? samples/Makefile
? samples/artprov/Makefile
? samples/calendar/Makefile
? samples/caret/Makefile
? samples/checklst/Makefile
? samples/config/Makefile
? samples/console/Makefile
? samples/controls/Makefile
? samples/dialogs/Makefile
? samples/dialup/Makefile
? samples/dnd/Makefile
? samples/docview/Makefile
? samples/docvwmdi/Makefile
? samples/dragimag/Makefile
? samples/drawing/Makefile
? samples/dynamic/Makefile
? samples/erase/Makefile
? samples/event/Makefile
? samples/exec/Makefile
? samples/font/Makefile
? samples/grid/Makefile
? samples/help/Makefile
? samples/htlbox/Makefile
? samples/html/Makefile
? samples/html/about/Makefile
? samples/html/help/Makefile
? samples/html/helpview/Makefile
? samples/html/printing/Makefile
? samples/html/test/Makefile
? samples/html/virtual/Makefile
? samples/html/widget/Makefile
? samples/html/zip/Makefile
? samples/image/Makefile
? samples/internat/Makefile
? samples/joytest/Makefile
? samples/layout/Makefile
? samples/listctrl/Makefile
? samples/mdi/Makefile
? samples/menu/Makefile
? samples/minifram/Makefile
? samples/minimal/Makefile
? samples/notebook/Makefile
? samples/png/Makefile
? samples/printing/Makefile
? samples/propsize/Makefile
? samples/render/Makefile
? samples/rotate/Makefile
? samples/sashtest/Makefile
? samples/scroll/Makefile
? samples/scrollsub/Makefile
? samples/shaped/Makefile
? samples/sockets/Makefile
? samples/splitter/Makefile
? samples/statbar/Makefile
? samples/taskbar/Makefile
? samples/text/Makefile
? samples/thread/Makefile
? samples/toolbar/Makefile
? samples/treectrl/Makefile
? samples/typetest/Makefile
? samples/validate/Makefile
? samples/widgets/Makefile
? samples/wizard/Makefile
? src/expat/Makefile
? src/expat/config.log
? src/expat/config.status
? src/expat/expat_config.h
? src/expat/libtool
? utils/Makefile
? utils/HelpGen/Makefile
? utils/HelpGen/src/Makefile
? utils/configtool/Makefile
? utils/configtool/src/Makefile
? utils/dialoged/Makefile
? utils/emulator/Makefile
? utils/emulator/src/Makefile
? utils/helpview/Makefile
? utils/helpview/src/Makefile
? utils/hhp2cached/Makefile
? utils/tex2rtf/Makefile
? utils/tex2rtf/src/Makefile
Index: src/generic/dirctrlg.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/src/generic/dirctrlg.cpp,v
retrieving revision 1.80
diff -b -u -2 -r1.80 dirctrlg.cpp
--- src/generic/dirctrlg.cpp	2003/12/20 15:13:15	1.80
+++ src/generic/dirctrlg.cpp	2003/12/27 23:19:29
@@ -835,4 +835,11 @@
             int style = wxDIR_FILES;
             if (m_showHidden) style |= wxDIR_HIDDEN;
+            // Process each filter (ex: "JPEG Files (*.jpg;*.jpeg)|*.jpg;*.jpeg")
+            wxStringTokenizer strTok;
+            wxString curFilter;
+            strTok.SetString(m_currentFilterStr,wxT(";"));
+            while(strTok.HasMoreTokens())
+            {
+                curFilter = strTok.GetNextToken();
             if (d.GetFirst(& eachFilename, m_currentFilterStr, style))
             {
@@ -845,4 +852,5 @@
                 }
                 while (d.GetNext(& eachFilename));
+                }
             }
         }
Index: utils/dialoged/src/Makefile
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/utils/dialoged/src/Makefile,v
retrieving revision 1.8
diff -b -u -2 -r1.8 Makefile
--- utils/dialoged/src/Makefile	2002/02/20 23:19:40	1.8
+++ utils/dialoged/src/Makefile	2003/12/27 23:19:35
@@ -1,3 +1,2 @@
-# Generated automatically from Makefile.in by configure.
 #
 # File:		makefile.unx
@@ -11,7 +10,9 @@
 # Makefile for Tex2RTF (Unix)
 
-top_srcdir = ../../..
+top_srcdir = ../../../..
 top_builddir = ../../..
 program_dir = utils/dialoged/src
+APPEXTRALIBS=$(top_builddir)/lib/lib@WX_LIBRARY_BASENAME@_wxdeprecated-@WX_RELEASE@.so
+APPEXTRADEFS=-I$(top_srcdir)/contrib/include
 
 PROGRAM=DialogEd

 	  	 
